### PR TITLE
9th docker local run instructions

### PR DESCRIPTION
**Pull request by red1408**
Needs 7th and 8th.

### CODE REVIEW

**Comment by danielbeaudreau** [comment]
I've created a wiki page, can you please add a page with these instructions to the wiki? For reference here's some example pages in the Dicom fuse repo: https://github.com/GoogleCloudPlatform/healthcare-api-dicom-fuse/wiki

We can then add a link to the wiki in the readme

**Comment by red1408** [comment]
Added the wiki page for this.
Do I leave this as is in readme, or just refer to wiki? If second, then on per case basis or simply something along the lines of 'for further questions check wiki'?

File: README.md [hunk 1]
```diff
@@ -235,3 +243,35 @@ TAG=gcr.io/${PROJECT}/dicom-export-adapter
...
+Solution is to mount local credentials file as a volume:
+
+```shell
+-v /path/to/google_credentials.json:/credentials -e GOOGLE_APPLICATION_CREDENTIALS='/credentials'
+```
+
+Import adapter also needs AET dictionary json to support C-MOVE and Storage Commitment service:
+
+```shell
+-v /path/to/HealthcareProjects/AETs.json:/AETs
```
Comment (danielbeaudreau) [inline README.md]: I think we could remove HealthcareProjects from the path and just have path to AETs.json similar to credentials json?

Comment (red1408) [reply README.md]: Yes, missed it. Fixed in wiki.

**Comment by red1408** [comment]
Note, this pull-request also serves as merge between 7th and 8th. Tried to do these independently this time. Result is imo too much error-prone merging work, I'll keep further pull-request sequential as before.

File: README.md [hunk 2]
```diff
@@ -235,3 +240,5 @@ TAG=gcr.io/${PROJECT}/dicom-export-adapter
 gradle dockerBuildImage -Pdocker_tag=${TAG}
 docker push ${TAG}
 ```
+
+Check [wiki](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/wiki/Running-Docker-image-locally) for instructions on how to run Import Adapter Docker image locally.
```
Comment (danielbeaudreau) [inline README.md]: Minor edit: "Instructions on how to run the Import Adapter Docker image locally are available on the [wiki.](...)"

Comment (red1408) [reply README.md]: Ok

**Comment by danielbeaudreau** [comment]
Can you please rebase this change? I think it's just a single sentence added to the readme. Thanks

**Comment by red1408** [comment]
Done

**Pull request merged**